nym-node/src/node/metrics/handler/client_sessions.rs

```diff
@@ -73,6 +73,14 @@ impl GatewaySessionStatsHandler {
         Ok(())
     }
 
+    async fn handle_session_delete(
+        &mut self,
+        client: DestinationAddressBytes,
+    ) -> Result<(), StatsStorageError> {
+        self.storage.delete_active_session(client).await?;
```

Comment:
Can you delete the entry from `sessions_unique_users` as well? (For the correct day)

Reply:
I think we can assume that we won't be duplicating identities in the span of a few days, if we are we should fix that... Otherwise added

Reply:
It's only a problem if we're deleting actual clients and that would mean we were able to generate the same keypair as somebody. It's fair to say it's not gonna happen, so we're good